DOAP.rdf

```diff
@@ -51,7 +51,7 @@
         <asfext:title>GeoAPI</asfext:title>
         <asfext:body>Open Geospatial Consortium</asfext:body>
         <asfext:id>OGC 09-083r3</asfext:id>
-        <asfext:url rdf:resource = "https://www.ogc.org/standards/geoapi"/>
+        <asfext:url rdf:resource = "http://www.opengeospatial.org/standards/geoapi"/>
```

Comment:
I don't recall why this value changed. Maybe I got the wrong file, or it changed after I did the porting to Hugo @desruisseaux ?